cluster/src/fpindexcluster/proxy_service.py

```
@@ -0,0 +1,267 @@
+        # Index management
+        self.app.router.add_put("/{index}", self.create_index)
+        self.app.router.add_get("/{index}", self.get_index_info)
+        self.app.router.add_delete("/{index}", self.delete_index)
+        
+        # Single fingerprint operations
+        self.app.router.add_put("/{index}/{fp_id}", self.put_fingerprint)
+        self.app.router.add_get("/{index}/{fp_id}", self.get_fingerprint)
+        # HEAD is automatically handled by aiohttp for GET routes
+        self.app.router.add_delete("/{index}/{fp_id}", self.delete_fingerprint)
+    
+    async def start(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `start`

Add return type annotation: `None`

(ANN201)